host/class/msc/usb_host_msc/src/msc_scsi_bot.c

```diff
@@ -68,6 +69,21 @@ static const char *TAG = "USB_MSC_SCSI";
 
 #define CWB_FLAG_DIRECTION_IN (1<<7) // device -> host
 
+/**
+ * @brief Sense keys list
+ *
+ * @see USB Mass Storage Class – UFI Command Specification, Revision 1.0
+ * Table 51 - Sense Keys, ASC/ASCQ Listing for All Commands (sorted by Key)
+ *
+ */
+typedef enum {
+    SENSE_WRITE_PROTECTED_MEDIA = 0x00072700,
```

Comment:
Nitpick: Maybe it make sense to split it into three enums: Sense key,  ASC and ASCQ or to create a struct of REQUEST_SENSE reply. 
Something like: 

```
struct request_sense_data_t {
    union { 
        struct { 
            uint8_t err_code:   7;
            uint8_t valid:      1;
        }; 
        uint8_t byte0;
    };
    uint8_t byte1_reserved;
    union {
        struct {
            uint8_t sense_key: 4;
            uint8_t reserved4: 4;
        };
        uint8_t byte2;
    };
    uint8_t info[4];
    uint8_t additional_sense_len;
    uint8_t byte8_11_reserved[4];
    uint8_t asc;
    uint8_t ascq;
    uint8_t byte14_17_reserved[4];
} __attribute__((packed));
```
For easier representation of return values and documentation matching
![image](https://github.com/espressif/esp-usb/assets/11456035/597d75dc-6e83-491f-9083-d8278bb43099)

Reply:
Such a struct is already implemented in the `msc_sci_bot.c` as Command Block Wrapper

```
typedef struct __attribute__((packed))
{
    uint8_t error_code;
    uint8_t reserved_0;
    uint8_t sense_key;
    uint32_t info;
    uint8_t sense_len;
    uint32_t reserved_1;
    uint8_t sense_code;
    uint8_t sense_code_qualifier;
    uint32_t reserved_2;
} cbw_sense_response_t;
```

But it's not entirely according to the documentation, unlike what you suggested. I think we can update the CBW structs to be exactly according to the docs, but it would make sense to update all the CBW structs, in different MR.